website/package.json

```diff
@@ -9,7 +9,8 @@
     "rename-version": "docusaurus-rename-version"
   },
   "devDependencies": {
-    "docusaurus": "1.0.15"
+    "docusaurus": "1.1.0",
+    "opn": "5.3.0"
```

Comment:
Hmm why do you need to add this? Are you getting an error? It should be installed as part of Docusaurus core.

Reply:
yes, agree

Reply:
My bad, it's a bug in Docusaurus. Submitted a fix for it in https://github.com/facebook/Docusaurus/pull/680